ompi/communicator/communicator.h

```diff
@@ -897,6 +901,15 @@ OMPI_DECLSPEC int ompi_comm_split_type(ompi_communicator_t *comm,
                                        struct opal_info_t *info,
                                        ompi_communicator_t** newcomm);
 
+/**
+ * Set newcomm's disjoint flag based on oldcomm if provided.
```

Comment:
This is not really what this functions does, because if the flag is not set in the parent communicator it then goes on and do a reduction.

Reply:
Enriched document in update.